_source/_posts/2024-02-29-3pc.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: blog_post
+title: "The End of Third-Party Cookies"
+author: edunham
+by: advocate
+communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
+description: ""
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image: blog/3pc/social.jpg
+type: awareness
+---
+
+## What are 3rd party cookies?
+
+Cookies are as old as the internet. Historically, cookies were among the only options for personalizing a user's online experience and carrying their preferences from page to page. First-party cookies are issued by the web site where they're used, and third-party cookies come from other domains. 
```

Comment:
nit: Change `third-party` to `3rd party` for consistency

Reply:
regex'd :)